plugins/gateway-backend/package.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "@backstage/plugin-gateway-backend",
+  "version": "1.0.0",
```

```suggestion
  "version": "0.0.0",
```

and add a changeset at minor level?

---

can we start from 1.0.0?

---

If you explicitly want to start at 1.0 without room for letting it mature for a while at 0 - sure, it's a choice we can make. We've traditionally always started at 0.1.0 but it's a matter of taste and how secure you feel about probably not having to make breaking changes to the interface